cmd/rule-evaluator/main.go

```diff
@@ -392,9 +393,16 @@ func main() {
 			}
 		})
 
+		// https://prometheus.io/docs/prometheus/latest/querying/api/#build-information
+		buildInfoHandler := promapi.BuildinfoHandlerFunc(log.With(logger, "handler", "buildinfo"), "rule-evaluator", version)
```

Comment:
We might need to stick to Prometheus versions here to mimic this API - it essentially tells what version of Prometheus we are compatible with. But if that works as well, we can iterate later (: